models/models.py

```diff
@@ -0,0 +1,85 @@
+from django.db import models
+from django.db.models.deletion import CASCADE
+
+# Create your models here.
+
+class User(models.Model):
+    name      = models.CharField(max_length=100)
+    email     = models.CharField(max_length=100)
+    password  = models.CharField(max_length=200)
+    contact   = models.CharField(max_length=100)
+    address   = models.CharField(max_length=100)
+    create_at = models.DateTimeField(auto_now_add=True)
+    update_at = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        db_table='users'
+
+
+class Product(models.Model):
+    name              = models.CharField(max_length=100)
+    price             = models.DecimalField(max_digits=10, decimal_places=2)
+    category          = models.ForeignKey('Products_Category',on_delete=CASCADE)
+    rating            = models.IntegerField()
+    image_description = models.URLField()
+    create_at         = models.DateTimeField(auto_now_add=True)
+    update_at         = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        db_table = 'products'
+
+class Products_Category(models.Model):
+    name = models.CharField(max_length=100)
+
+    class Meta:
+        db_table='product_category'
```

Comment:
class명은 camelCase를 사용해주세요!
```suggestion
class ProductCategory(models.Model):
    name = models.CharField(max_length=100)

    class Meta:
        db_table='product_category'
```

Reply:
상품안에 카테고리가 하나이기 때문에 Category로 수정하였습니다